component.css

```diff
@@ -0,0 +1,183 @@
+@import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Sans:ital,wght@0,100;0,200;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,400;1,500;1,600;1,700&family=Nunito&family=Roboto:wght@100;300;400;500;700&display=swap');
+@import url('component\\card\\card.css') ;
+@import url('component\\Badges\\badge.css') ;
+:root{
+    --black:rgb(20 20 20);
+    --red:#e50914;
+    --white:#fff;
+    --font:'Roboto', sans-serif;
+}
+
+*{
+    background-color: var(--black);
+    font-family: var(--font);
+    box-sizing: border-box;
+}
+.shadow{
+    box-shadow: 5px 5px 8px 0 grey;
+}
+h1,h2,h3,h4,h5,h6{
+    color: var(--white);
+}
+p{
+    color: var(--black);
```

Comment:
if you give html tags direct font color and then you use this library, chances are this font color pattern follows throughout. To ignore this behaviour at that time you need to reset the font color value to these tags (if you don't want font color as white there).